src/request/remove-key/index.html

```diff
@@ -72,28 +74,57 @@
+                    </div>
+                    <div>
+                        <div data-i18n="remove-key-login-file-question">
+                            Is your LoginFile savely stored and accessible?
+                        </div>
+                        <button id="show-download-login-file" data-i18n="remove-key-download-login-file"
+                            class="nq-button-s">
+                            Download LoginFile
+                        </button>
+                    </div>
+                </div>
+                <div class="backup-option">
+                    <img src="../../assets/sample-recovery-words.png" alt="sample recovery words" />
+                    <div>
+                        <div data-i18n="remove-key-recovery-words-question">
+                            Do you know where your Recovery Words are?
+                        </div>
+                        <button id="show-recovery-words" data-i18n="remove-key-show-recovery-words" class="nq-button-s">
+                            Create a backup
+                        </button>
+                    </div>
                 </div>
-                <div class="download-key-file"></div>
-                <div class="flex-grow"></div>
             </div>
 
-            <div class="page-footer hide-for-download">
-                <form class="passphrase-box"></form>
+            <div class="flex-grow"></div>
+
+            <div class="page-footer nq-card-footer">
+                <button id="remove-key-first-confirm" class="hide-for-final-confirm nq-button red"
```

Comment:
This probably does not look like it is supposed to look?

Reply:
Right. Soeren and me weren't quite happy with the 'checkbox-button' style. There will be another solution probably.